templates/base.html.twig

```diff
@@ -19,6 +19,7 @@
         {% endblock %}
         {% block javascripts %}
             {{ encore_entry_script_tags('js/app') }}
+            {{ encore_entry_script_tags('js/urlBuilder') }}
```

Comment:
Let's include the file urlBuilder in the app file to avoir aving many encore entries since it's always required